.circleci/config.yml

```diff
@@ -84,6 +84,25 @@ jobs:
             - run: yarn wsrun test:circleci @0x/contracts-exchange
             - run: yarn wsrun test:circleci @0x/contracts-exchange-forwarder
             - run: yarn wsrun test:circleci @0x/contracts-coordinator
+    test-contracts-ganache-3.0:
+        docker:
+            - image: circleci/node:9-browsers
+        working_directory: ~/repo
+        steps:
+            - restore_cache:
+                  keys:
+                      - repo-{{ .Environment.CIRCLE_SHA1 }}
+            - run: yarn wsrun test:circleci @0x/contracts-multisig
+            - run: yarn wsrun test:circleci @0x/contracts-utils
+            - run: yarn wsrun test:circleci @0x/contracts-exchange-libs
+            - run: yarn wsrun test:circleci @0x/contracts-erc20
+            - run: yarn wsrun test:circleci @0x/contracts-erc721
+            - run: yarn wsrun test:circleci @0x/contracts-erc1155
+            # - run: yarn wsrun test:circleci @0x/contracts-extensions
+            - run: yarn wsrun test:circleci @0x/contracts-asset-proxy
+            - run: yarn wsrun test:circleci @0x/contracts-exchange
+            # - run: yarn wsrun test:circleci @0x/contracts-exchange-forwarder
```

Comment:
Please add `# TODO()` commands next to these explaining why they are commented out (for posterity).